Options.py

```diff
@@ -1198,15 +1198,20 @@ def dictify_range(option: Range):
 
     for game_name, world in AutoWorldRegister.world_types.items():
         if not world.hidden or generate_hidden:
-            all_options: typing.Dict[str, AssembleOptions] = {
-                option_name: option for option_name, option in world.options_dataclass.type_hints.items()
-                if option.visibility & Visibility.template
-            }
+
+            option_groups = {option: option_group.name
+                             for option_group in world.web.option_groups
+                             for option in option_group.options}
+            ordered_groups = ["Game Options"]
+            [ordered_groups.append(group) for group in option_groups.values() if group not in ordered_groups]
+            grouped_options = {group: {} for group in ordered_groups}
+            for option_name, option in world.options_dataclass.type_hints.items():
+                grouped_options[option_groups.get(option, "Game Options")][option_name] = option
```

Comment:
This appears to remove the ability to hide options from the template yaml, so games relying on that (LttP) currently break.

Reply:
Good find. This has been corrected.